<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>259</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>맑은 고딕</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Statics</string>
  </property>
  <widget class="QComboBox" name="cbMydeck">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>10</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>내 덱</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>VS</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbOppocls">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>60</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>상대 직업</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbOppodeck">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>110</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>상대 덱</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tablesta">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>161</width>
     <height>115</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>false</bool>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="rowCount">
    <number>3</number>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>38</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <row>
    <property name="text">
     <string>선공</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>후공</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>합계</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>승</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>패</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>승률</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>501</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="staticLayout"/>
  </widget>
  <widget class="QLabel" name="lb_result">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>461</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>55</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Result</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
